src/utils.c

```diff
@@ -20,84 +20,56 @@ bool is_bool(SEXP x) {
  * Dispatch with two arguments
  *
  * @param fn The method to call.
- * @param x,y,z Arguments passed to the method.
- * @param fn_sym,x_sym,y_sym,z_sym Symbols to which `x`, `y` and `z`
- *   should be assigned.  The assignment occurs in `env` and the
- *   dispatch call refers to these symbols.
+ * @param syms Symbols to which arguments are assigned.  The
```

Comment:
What is `syms`? A null-terminated array?

Reply:
ah good point, I'll make this more precise.